automatic/octave.portable/update.ps1

```diff
@@ -6,30 +6,22 @@ Import-Module AU
 $releases = "https://ftp.gnu.org/gnu/octave/windows/?C=M;O=D"
 
 function global:au_BeforeUpdate {
-  $checksumType = 'sha256'
-  $Latest.ChecksumType64 = $Latest.ChecksumType32 = $checksumType
-
-  $Latest.Checksum32 = Get-RemoteChecksum $Latest.URL32 -Algorithm $checksumType
-  $Latest.Checksum64 = Get-RemoteChecksum $Latest.URL64 -Algorithm $checksumType
+  $Latest.ChecksumType64 = 'sha256'
```

Comment:
As we're using `$Latest.checksumType64` as the `-Algorithm` in the call to `Get-RemoteChecksum`, we shouldn't hard-code this here but instead use `$Latest.checksumType64`,